utils/scale-ci-diagnosis/ocp_diagnosis.sh

```diff
@@ -130,13 +120,16 @@ function store() {
 	if [[ -z $STORAGE_MODE ]]; then
 		echo "Looks like STORAGE_MODE is not defined, storing the results on local file system"
 		$1;
-	elif [[ $STORAGE_MODE == "pbench" ]]; then
-		set_pbench;
+	elif [[ $STORAGE_MODE == "snappy" ]]; then
 		$1;
-	elif [[ $STORAGE_MODE == "data_server" ]]; then
-		snappy script-login
-		$1 && snappy post-file "$OUTPUT_DIR/$2" --filedir $SNAPPY_FILE_DIR
-		snappy logout
+		echo -e "snappy server as backup enabled"
+		source ../snappy-move-results/common.sh
+		export snappy_path="$SNAPPY_USER_FOLDER/$runid$platform-$cluster_version-$network_type/$workload/$folder_date_time/"
```

Comment:
$runid$platform --> $runid_$platform ?

Reply:
this is taken care in the common.sh , so that if there's a manual run the _ will be avoided , but if runid is set then in common.sh it becomes $runid_